helm/deploy_test.go

```diff
@@ -132,6 +137,21 @@ func TestHelmDeployConfigureUndeploy(t *testing.T) {
 
 	// Check that the rendered helm env file works
 	validateHelmEnvFile(t, testServiceAccountKubectlOptions)
+
+	// Revoke the tiller service account
+	rbacGroups := []string{}
+	rbacUsers := []string{}
+	serviceAccounts := []string{fmt.Sprintf("%s/%s", namespaceName, testServiceAccountName)}
+	assert.NoError(t, RevokeAccess(kubectlOptions, namespaceName, rbacGroups, rbacUsers, serviceAccounts))
+
+	// No role for service account
+	assert.Error(t, validateNoRole(t, kubeClient, namespaceName, testServiceAccountName))
```

Comment:
The one issue I see with this is that you're assuming a certain type of error will be returned: role does not exist. But if a different error occurs here (e.g. "permissions denied") the test won't distinguish between the two.

Reply:
I'm assuming that I cannot get the role (or role binding, or secret; more or less the same approach for each of these tests). If I can get the role, then something went wrong on the revocation. If there was a permissions problem or otherwise, it would've been thrown in `RevokeAccess()`.

Reply:
If you take the updated labels approach I proposed here (https://github.com/gruntwork-io/kubergrunt/pull/50#discussion_r286278999), you can use the `List` interface instead for a more targeted existence check.